ffi/blitbuffer.lua

```diff
@@ -24,6 +24,17 @@ local uint16pt = ffi.typeof("uint16_t*")
 local uint8pt = ffi.typeof("uint8_t*")
 local posix = require("ffi/posix_h") -- luacheck: ignore 211
 
+local _ = require("ffi/turbojpeg_h")
+local turbojpeg
+if ffi.os == "Windows" then
+    turbojpeg = ffi.load("libs/libturbojpeg.dll")
```

Comment:
The libraries are named the same. 

```lua
local turbojpeg = ffi.load("turbojpeg")
```
should work.

Reply:
I have copied this from `pic.lua`, so should it be changed there too?

Reply:
> I have copied this from pic.lua, so should it be changed there too?

If it works, yeah. But please keep it in a different PR.

At the end `libs` is part of cpath and we shouldn't hardcode the full so name if there's no reason to. It makes some things more difficult. For instance: https://github.com/koreader/koreader-base/issues/1237

Sadly it is used on a bunch of files: https://github.com/search?q=org%3Akoreader+ffi.load%28%22libs&type=code. Also replacement is not as easier as it seems, because we break expectations on some libraries (for example we built some of them as .so in osx, where the OS expect dylib)

But for new cases it is indeed better (if it works :p)

Reply:
> If it works, yeah. But please keep it in a different PR.

^ That. I imagine pic.lua predates the cpath changes or some such.